misc/images/materialized/Dockerfile

```diff
@@ -21,8 +21,9 @@ RUN apt-get update \
     && groupadd --system --gid=999 materialize \
     && useradd --system --gid=999 --uid=999 --create-home materialize \
     && mkdir /mzdata \
+    && mkdir /prometheus \
     && mkdir /cockroach-data \
-    && chown materialize /mzdata /cockroach-data
+    && chown materialize /mzdata /prometheus /cockroach-data
```

Comment:
This image is meant to a publicly consumable artifact and I'd like to keep it clean of mzcompose cruft! I think we can do this in a way that does not require changes to the Docker image. See below.